style.css

```diff
@@ -2646,6 +2646,44 @@ div.comment:first-of-type {
 /*	13. Site Pagination
 /* --------------------------------------------------------------------------------------------- */
 
+.pagination .nav-links {
+	display: flex;
+	flex-wrap: wrap;
+	font-size: 1.8rem;
+	padding-top: 3rem;
+	font-weight: 600;
+}
+
+.pagination a {
+	text-decoration: none;
+	margin: 0 2rem 2rem 2rem;
+}
+
+.pagination a:hover {
+	text-decoration: underline;
+}
+
+.pagination .current {
+	margin: 0 2rem 2rem 2rem;
+}
+
+.pagination .dots {
+	display: inline-block;
+	vertical-align: middle;
```

Comment:
Removing the `display: inline-block;`and `vertical-align: middle;`, and moving the `margin-top: -.3em` CSS that you have in the media query CSS to the base, seems to position the ellipsis correctly in all screen sizes for me.